giza/giza/content/options/models.py

```diff
@@ -71,6 +71,20 @@ def optional(self, value):
             raise TypeError
 
     @property
+    def command(self):
+        if 'command' in self.state:
+            return self.state['command']
+        else:
+            return 
+
+    @command.setter
+    def command(self, value):
+        if isinstance(value, basestring):
+            self.state['command'] = value
+        else:
+            raise TypeError
```

Comment:
TypeError should be instantiated with an argument explaining what's going on here